website/gridscale.erb

```diff
@@ -77,10 +77,12 @@
               <a href="/docs/providers/gridscale/r/network.html">gridscale_network</a>
             </li>
             <li<%= sidebar_current("docs-gridscale-resource-paas") %>>
-              <a href="/docs/providers/gridscale/r/securityzone.html">gridscale_paas</a>
+              <a href="/docs/providers/gridscale/r/paas.html">gridscale_
```

Comment:
this looks like the "paas" in the text for the link got botched.

Reply:
Should be resolved :wink:

Reply:
:)